README.md

```diff
@@ -1,56 +1,25 @@
-
-De definitie van een data space is minder duidelijk. [Gaia-X](https://gaia-x.eu/what-is-gaia-x/core-elements/data-spaces/) heeft het bijvoorbeeld over 
+De Nederlandse overheid heeft de [Werkagenda Waardengedreven Digitaliseren](https://www.digitaleoverheid.nl/kabinetsbeleid-digitalisering/werkagenda/) opgesteld. Eén van de programmalijnen luidt: "Iedereen heeft regie op het digitale leven." *Daarvoor is regelgeving en beleid nodig, rond wallets en andere basisvoorzieningen.* [Solid](https://solidproject.org/) is een specificatie die het mogelijk maakt om je data op een veilige manier, decentraal, op te slaan in PODs, Personal Online Datastores. Als Kadaster bereiden wij ons voor op deze ontwikkelingen. Het begrijpen van de technologie is hiervoor randvoorwaardelijk. Daarom bouwen wij aan een demonstrator, met als voorbeeld usecase [Zorgeloos Vastgoed](https://www.zorgeloosvastgoed.nl/), om de (on)mogelijkheden van SOLID pods te onderzoeken en begrijpen.
 
-> "The term ‘data space’ refers to a type of data relationship between trusted partners who adhere to the same high level standards and guidelines in relation to data storage and sharing within one or many Vertical Ecosystems." 
+Het is een (emerging) technologie waar we nog geen of nauwelijks ervaring mee hebben en ook benieuwd zijn hoe deze zich verhoudt tot andere ontwikkelingen:
+-	Verifiable Credentials
+-	Linked data 
+-	Wallets zoals de Europese Digitale Identiteit 
+-	Blockchain 
+-	Data Spaces 
 
-Onze vertaling is dat pods vooral gericht zijn op persoonlijke data kluizen; een persoonlijke online datastore. Als burger heb ik zelf controle over wat ik daar in zet, wie of welke apps ik toegang verleen en welke data ik (dus) deel met wie. De visie van het Solid Project gaat nog verder in de zin dat _alle_ data die ik online deel, in mijn pod zou moeten staan. Applicaties en platformen kunnen deze data gebruiken zolang zij toegang verleent krijgen door de burger zelf. Platformen als Facebook, Twitter e.d. zijn rijke applicaties die vooral bestaan om de functionaliteiten die zij bieden, terwijl de data in alle pods van alle gebruikers staat. (Dit is ook wat [Post Platforms Initiative](https://postplatforms.org/#concept) tracht te bereiken)
+#DataMinimalisatie #RegieOpGegevens #PrivacyEnhancingTechnologies #SOLID #VerifiableCredentials #EmergingTech #Cloud #Security #LinkedData 
 
-Data spaces lijken meer gericht te zijn op organisaties met als doel het stimuleren en bereiken van data delen/samenwerken en tegelijkertijd controle te houden over je data.
 
-Binnen dit project zal de focus meer liggen op data pods.
+## Table of Contents
 
-### WebID
+- [References](References.md) - een overzicht van interessante links
+- [Background](Background.md) - een overzicht van onze opgedane kennis rondom data pods, WebID, VCs etc.
```

Comment:
Zou het iets toevoegen om de ToC (Table of Contents) van de Background hier ook al (als sub list) toe te voegen? Dan kun je vanaf hier direct naar de betreffende onderwerpen springen ... ? 🤔

Reply:
Done